.meteor/packages

```diff
@@ -97,3 +97,4 @@ webapp-hashing@1.0.9
 rocketchat:oauth2-server
 rocketchat:i18n
 dandv:caret-position
+twbs:bootstrap
```

Comment:
Why do you need this?

Reply:
I have used glyphicon in search option that is why i have added this package

Reply:
We have all the resources we need, natively.
Please, remove this dependency, thanks.

Reply:
Removed

Reply:
No, it's still there.